packages/debugger/lib/txlog/actions/index.js

```diff
@@ -211,6 +211,19 @@ export function logEvent(pointer, newPointer, step, decoding, rawEventInfo) {
   };
 }
 
+//this may be replaced once decoding info was added
```

Comment:
This comment is confusing.

Reply:
Oh, oops, incorrect tense on that verb (should be "is added")... is that the confusing part?  I'll go fix that...